src/test/java/net/snowflake/client/jdbc/DatabaseMetaDataLatestIT.java

```diff
@@ -165,6 +167,8 @@ public void testUseConnectionCtx() throws SQLException {
 
       resultSet = databaseMetaData.getCrossReference(null, null, null, null, null, null);
       assertThat(getSizeOfResultSet(resultSet), greaterThanOrEqualTo(2));
+
+      connection.createStatement().execute("drop schema " + customSchema);
```

Comment:
Maybe we should have it `finally`/some cleanup method? This schema won't be dropped by the cleanup script, so if it fails on CI, it will stay forever?